### ISSUE TITLE
When running in Wine, the filebrowser for saving & opening files is not drawn.

### ISSUE THREAD

**Issue by r4dian** [Dn-Programming-Core-Management/Dn-FamiTracker#55]
Set up:
The 64bit 0.3.0.0 version of Dn-Famitracker, (debug or release)
MacOS Catalina 10.15.7
Wine 5.7 installed via homebrew (wine-stable or wine-devel package is the same result)

Steps to reproduce:

- Open the `File, open....` menu or use the icon
- Open the `File, Save` or `File, Save As...` menu or use the icon
- Give the file a name and save it
- Open the `File, open....` menu and type the same filename

Results:

You can't tell what folder you're in or navigate to another.
The file is saved (appears on the Mac desktop by default, nice enough)
Files can be opened successfully only from the desktop, and by entering the names, not by selection.
As a workaround I'm able to drag & drop files to open.

The following appears in the terminal from which Wine was launched:

```sh
002e:fixme:commdlg:IServiceProvider_fnQueryService Interface {e07010ec-bc17-44c0-97b0-46c7c95b9edc} requested from unknown service {e07010ec-bc17-44c0-97b0-46c7c95b9edc}
002e:fixme:shell:ViewModeToListStyle ViewMode 0 not implemented
002e:fixme:shell:IShellBrowser_fnSendControlMsg stub, 0x2632630 (2, 1026, a003, 0, 0x50b5d0)
002e:fixme:shell:IShellBrowser_fnSendControlMsg stub, 0x2632630 (2, 1026, a004, 0, 0x50b5d0)
002e:fixme:shell:IShellBrowser_fnSendControlMsg stub, 0x2632630 (2, 1025, a003, 1, 0x50b5d0)
002e:fixme:shell:IShellBrowser_fnSendControlMsg stub, 0x2632630 (2, 1025, a004, 1, 0x50b5d0) 
```

<img width="710" alt="Screenshot 2021-04-07 at 16 11 54" src="https://user-images.githubusercontent.com/1264174/113891594-19a83800-97bd-11eb-94c4-29bf1736bcc9.png">
<img width="768" alt="Screenshot 2021-04-07 at 16 11 33" src="https://user-images.githubusercontent.com/1264174/113891608-1c0a9200-97bd-11eb-9df6-a1f9f1ee12f0.png">

**Comment by nyanpasu64** [comment]
i have some theories:

- use-after-free when passing a filename into HistoryFileDialog. i don't think it actually results in a bug since CString is refcounted and reuses the underlying memory (which doesn't get invalidated when a CString copy is destroyed, but idk.
- wine's vista-style dialogs are poorly implemented, so openmpt uses xp-style dialogs on wine since they actually work, but famitracker doesn't.

can't debug since i don't have mac or wine-on-mac.

**Comment by r4dian** [comment]
Your theory was enough to resolve the issue for me by changing to use Windows XP as the Windows Version for Dn-Famitracker using `winecfg`.

This does not resolve issue #54 though.

**Comment by r4dian** [comment]
This is resolved in `wine-6.0.2`
